Integrations/ThreatConnect/ThreatConnect.py

```diff
@@ -0,0 +1,1338 @@
+    args = demisto.args()
+    group_id = int(args['groupID'])
+    group_type = args['type']
+
+    success = tc_delete_group(group_id, group_type.lower())
+    if success:
+        demisto.results({
+            'Type': entryTypes['note'],
+            'ContentsFormat': formats['text'],
+            'Contents': '{} {} deleted Successfully'.format(group_type.lower(), group_id),
+        })
+    else:
+        return_error('Failed to delete {} {}'.format(group_type, group_id))
+
+
+def tc_delete_group(group_id, group_type):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('DELETE')
+    ro.set_request_uri('/v2/groups/{}/{}'.format(group_type, group_id))
+    response = tc.api_request(ro).json()
+
+    return response['status'] == 'Success'
+
+
+def tc_add_group_attribute_request(group_type, group_id, attribute_type, attribute_value):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('POST')
+    ro.set_request_uri('/v2/groups/{}/{}/attributes'.format(group_type, group_id))
+    body = {
+        'type': attribute_type,
+        'value': attribute_value,
+        'displayed': 'true'
+    }
+    ro.set_body(json.dumps(body))
+    response = tc.api_request(ro).json()
+
+    return response
+
+
+def tc_add_group_attribute():
+    group_id = int(demisto.args().get('group_id'))
+    group_type = demisto.args().get('group_type')
+    attribute_type = demisto.args().get('attribute_type')
+    attribute_value = demisto.args().get('attribute_value')
+    headers = ['Type', 'Value', 'ID', 'DateAdded', 'LastModified']
+    attribute = tc_add_group_attribute_request(group_type, group_id, attribute_type, attribute_value)
+    data = attribute.get('data').get('attribute')
+    contents = {
+        'Type': data.get('type'),
+        'Value': data.get('value'),
+        'ID': data.get('id'),
+        'DateAdded': data.get('dateAdded'),
+        'LastModified': data.get('lastModified')
+    }
+    context = {
+        'TC.Group(val.ID && val.ID === obj.ID)': contents
+    }
+
+    return_outputs(tableToMarkdown('The attribute was added successfully to group {}'.format(group_id),
+                                   contents, headers, removeNull=True), context, attribute)
+
+
+def add_group_security_label_request(group_type, group_id, security_label):
+    tc = get_client()
+    ro = RequestObject()
+    ro.set_http_method('POST')
+    ro.set_request_uri('/v2/groups/{}/{}/securityLabels/{}'.format(group_type, group_id, security_label))
+
+    response = tc.api_request(ro).json()
+
+    return response.get('status') == 'Success'
+
+
+def add_group_security_label():
+    group_id = int(demisto.args().get('group_id'))
+    group_type = demisto.args().get('group_type')
+    security_label = demisto.args().get('security_label_name')
+
+    add_group_security_label_request(group_type, group_id, security_label)
+
+    demisto.results('The security label {} was added successfully to {} {}'.format(security_label, group_type,
```

Comment:
need to improve the message a bit, like add something before group type of group id

Reply:
?

Reply:
Waiting for @kirbles19